extensions/amp-carousel/0.1/base-carousel.js

```diff
@@ -68,6 +68,8 @@ export class BaseCarousel extends AMP.BaseElement {
     this.prevButton_ = this.element.ownerDocument.createElement('div');
     this.prevButton_.classList.add('amp-carousel-button');
     this.prevButton_.classList.add('amp-carousel-button-prev');
+    this.prevButton_.classList.toggle(
```

Comment:
I think this needs to get added after the "hint" animation is finished, otherwise it is removing the "hint" where controls show up for a bit and then fade away.